theano/tensor/opt.py

```diff
@@ -1633,6 +1631,8 @@ def local_subtensor_make_vector(node):
                     ret = [x.owner.inputs[v]]
                 except IndexError:
                     raise NotScalarConstantError("Bad user graph!")
+
+                # Copy over stack trace from previous output to new output
```

Comment:
If I understand correctly, `ret` is a variable that was already in the graph, so we do not have to add a new stack trace. In fact, the code is not doing it, so I think we should remove that comment.